test/integration/cluster_controller_test.go

```diff
@@ -1,14 +1,15 @@
 // Copyright (C) 2017 ScyllaDB
 
-package cluster
+package integration
```

Comment:
these table tests are unit tests, they don't require integration env.

Reply:
yeah, ginkgo for units probably got me off track

Reply:
actually what would run a ginkgo framework in regular unit tests? ginkgo needs a main function to trigger the suite

Reply:
it does look like a great fit for standart golang table driven test, not sure why ginkgo is mixed there